protostar/starknet/cheatable_execute_entry_point.py

```diff
@@ -246,6 +240,12 @@ def _run(  # type: ignore
         assert isinstance(args_ptr, RelocatableValue)  # Downcast.
         runner.mark_as_accessed(address=args_ptr, size=len(entry_points_args))
 
+        if self.profiling:
+            self.append_runtime_profile(runner, contract_class, entry_point)
+            self.pop_contract_callstack()
+            if not CheatableExecuteEntryPoint.contract_callstack:
+                merge_and_save(CheatableExecuteEntryPoint.samples)
+
```

Comment:
I'd be nice to mark the modified code with `#region`.
https://marketplace.visualstudio.com/items?itemName=maptz.regionfolder